### DISCUSSION TITLE
Which is the database Verdaccio using?

### DISCUSSION THREAD

**Discussion by DannySu09** [verdaccio/verdaccio#2048] [Q&A] [answered]
Been thinking about migrating from [Cnpm](https://github.com/cnpm/cnpmjs.org) to Verdaccio. [Cnpm](https://github.com/cnpm/cnpmjs.org) is using **sqlite3** to store package meta data. How about Verdaccio?

**Comment by juanpicado** [comment, marked as answer]
By default nothing, the file system and a JSON file to define which packages are private, is really simple. But you can use different storage if you like.
